libs/langchain/langchain/chat_models/cohere.py

```diff
@@ -74,13 +74,32 @@ def _identifying_params(self) -> Dict[str, Any]:
         return {**{"model": self.model}, **self._default_params}
 
     def get_cohere_chat_request(
-        self, messages: List[BaseMessage], **kwargs: Any
+        self,
+        messages: List[BaseMessage],
+        connectors: List[Dict[str, str]],
```

Comment:
what're connectors and are they supposed to be required? if so, should we be passing them in in L114, 131, etc

Reply:
Updated the comment with context. They provide the llm with sources to search. Currently only web-search connector is supported and I think even when more are supported, they likely will only need to be set in the constructor.